### ISSUE TITLE
Multiple plays being marked for one show on Trakt

### ISSUE THREAD

**Issue by crxssed7** [michaldrabik/showly#738]
**Describe the bug**
Around 5 months ago I had noticed that my account had over 700 plays per episode for "The Office". At first I believed that this was a bug with Trakt as the only app I had connected to my account was Showly. So I manually removed the duplicate plays and submitted a support ticket. They informed me that the duplicate plays were in fact coming from Showly and that I should revoke the app. Ticket number #20980:
 
![image](https://github.com/michaldrabik/showly-2.0/assets/48864420/a7d1689e-39b9-4caf-8fbf-4501d85ba314)

I still wasn't convinced though that Showly was causing this as I believed it only supported single plays, so I didn't think much of it but I did revoke Showly and reauthenticate just to be safe.

The issue did not occur again until two weeks ago when I found the same thing happening to another show - "Initial D". Again I manually removed the duplicated plays but this time I didn't revoke Showly. Instead I disabled the automated sync and even the export to Trakt option. Just this morning I noticed it had occurred again (on "Initial D" again), but there weren't as many duplicates. This lead me to believe that maybe it was Showly adding duplicates whenever the sync ran; because automated sync was disabled I only synced manually and not as often (I've been using the official app for the past week to test things). To prove this to myself I removed all duplicate plays again and waited about 30 minutes before refreshing the page - there were no duplicates. I then ran a sync on Showly expecting it to not export any duplicates, but after the sync I refreshed the page and found 1 duplicate per episode. I tried this a few times but every time I ran a sync new duplicates were added, so at this point I'm convinced it's a bug with Showly.

This sounds like a bug that might be a difficult fix, so I've tried to give as much information as I have but let me know if you need anything else.

**Expected behavior**
I don't expect there to be duplicates everytime I sync.

**Comment by michaldrabik** [comment]
Hello and thanks for reporting. This is actually an issue that I'm trying to solve but can't reproduce it myself. I heard it happens occasionally for some users.

Is this happening to you still right now with "Initial D" show?

**Comment by crxssed7** [comment]
Yes it still occurs, do you need anything from me? Logs etc?

**Comment by michaldrabik** [comment]
Anything you have can help! : )

I cant figure out why the app would send duplicates when each sync starts from fetching latest progress from Trakt and excluding already seen episodes.

**Comment by crxssed7** [comment]
I'm not too sure how to get the app logs, is there a way in Showly to do this?

One thing I failed to mention was that I had Trakt instant upload turned on as well as the automated sync, however I disabled them both and it still occurs so I'm not sure if that's very helpful.

Also wanted to correct myself, I just tested the sync with "Export to Trakt" turned off and the issue does not occur.

**Comment by michaldrabik** [comment]
Yeah with just Importing enabled there is no way any duplication can happen since nothing is sent from App to Trakt.

Perhaps you have some other device that you could install Showly on and see if you have the same issue with "clean" install I wonder. That could give some info already. Unfortunately there is no way to extract any kind of logs by the user from the app itself.

**Comment by michaldrabik** [comment]
Also - how many shows do you have in total in "My Shows" currently?

**Comment by crxssed7** [comment]
I have 157 shows in "My Shows"

I don't have a second device I can test on right now, if I can get my hands on one I'll test it out.

**Comment by michaldrabik** [comment]
If you are willing to temporarily share your Trakt account so I can debug please send me an email to showlyapp@gmail.com

Of course I'm not encouraging it but it would be very helpful If i was able to debug locally.

**Comment by crxssed7** [comment]
Just sent you an email
